src/project.test.js

```diff
@@ -1,82 +1,129 @@
-      {
-        forestEcoregion: '3',
-        additional: 'unknown',
-        heightLevel: 'OSA',
-        forestType: '59L',
-        slope: 'unknown',
-        tannenareal: 'unknown',
-        relief: 'unknown',
-      },
-      'OM',
-    ).forestType,
-  ).toBe('1h');
-});
-
 test('invalid location values', () => {
   expect(() =>
-    project({ forestType: '60*', forestEcoregion: 'fooBar' }, 'UM'),
+    project({ forestType: '60*', forestEcoregion: 'fooBar' }, '5'),
   ).toThrowError('fooBar for forestEcoregion is not valid.');
 
   expect(() =>
     project(
-      { forestType: '60*', forestEcoregion: '1', heightLevel: 'fooBar' },
-      'OM',
+      { forestType: '60*', forestEcoregion: '1', altitudinalZone: 'fooBar' },
+      '6',
     ),
-  ).toThrowError('fooBar for heightLevel is not valid.');
+  ).toThrowError('fooBar for altitudinalZone is not valid.');
 
   expect(() =>
     project(
       {
         forestEcoregion: '1',
-        heightLevel: 'HM',
+        altitudinalZone: '8',
         forestType: '55 collin',
       },
-      'C',
+      '2',
     ),
   ).toThrowError('55 collin for forestType is not valid.');
 });
 
-test('invalid target heightLevel', () => {
+test('invalid target altitudinalZone', () => {
   expect(() => project({}, 'fooBar')).toThrowError(
-    'fooBar for targetHeightLevel is not valid.',
+    'fooBar for target altitudinal zone is not valid.',
   );
 });
 
-test('missing projection for valid location and targetHeight', () => {
-  expect(() =>
+/* Test for output section */
+
+test('check for unknown as only available option', () => {
+  expect(
+    project(
+      {
+        forestEcoregion: '1',
+        altitudinalZone: '6',
+        forestType: '1h',
+        silverFirArea: 'unknown',
+      },
+      '5',
+    ).additional,
+  ).toBe('unknown');
+});
+
+test('check optional field with multiple values', () => {
+  expect(
     project(
       {
         forestEcoregion: '1',
-        additional: 'unknown',
-        heightLevel: 'OSA',
-        forestType: '58L',
-        slope: 'unknown',
-        tannenareal: 'unknown',
-        relief: 'unknown',
+        altitudinalZone: '9',
+        forestType: '60*',
+        silverFirArea: 'unknown',
       },
-      'HM',
-    ),
-  ).toThrowError('Found no projection for selected targetHeightLevel.');
+      '8',
+    ).slope,
+  ).toBe(undefined);
+});
+
+test('empty target for incomplete location values', () => {
+  expect(
+    project(
+      {
+        forestEcoregion: '1',
+        forestType: '60*',
+      },
+      '8',
+    ).target,
+  ).toBe(undefined);
+});
+
+test('option field with values for complete location values', () => {
+  expect(
+    project(
+      {
+        forestEcoregion: '1',
+        altitudinalZone: '9',
+        forestType: '60*',
+        silverFirArea: 'unknown',
+      },
+      '8',
+    ).options.slope,
+  ).toMatchObject(['<70', '>70']);
+});
+
+test('empty option field for incomplete location values', () => {
```

Comment:
Whats the difference to test("check optional field with multiple values") above?

Reply:
I removed this one, does not seem to have any difference with test('empty option field for incomplete location values')